examples/rvc-app/rvc-common/include/rvc-device.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "rvc-mode-delegates.h"
+#include "rvc-operational-state-delegate.h"
+#include <app/clusters/mode-base-server/mode-base-server.h>
+#include <app/clusters/operational-state-server/operational-state-server.h>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+
+const EndpointId RvcRunModeEndpoint          = 1;
```

Comment:
Wouldn't it make more sense to have the endpoint be a constructor argument to the RvcDevice class?

In particular, per the device type, all these clusters should be on the _same_ endpoint, so why the multiple constants?

Reply:
Fixed this.